### ISSUE TITLE
Feature request: export section tags with uniorg-rehype

### ISSUE THREAD

**Issue by ThomasFKJorna** [rasendubi/uniorg#20]
Atm uniorg creates an AST with 'sections', but uniorg-rehype is not able to convert this to section html elements, which would be useful for implementing collapsible headers org-style, see https://github.com/org-roam/org-roam-ui/issues/107#issue-1024077808.

I think this is due to this line which only exports the children of the section Object of the AST?
https://github.com/rasendubi/uniorg/blob/3d6bc077f2ee16eebf35d57e6b93d98022e0b65d/packages/uniorg-rehype/src/org-to-hast.ts#L109

I feel like providing an option which sets this to something like
```typescript
return h(org, 'section', {className: `section-level-${headline.level}`}, toHast(org.children))
```
should work, but I have not been able to modify uniorg in a sensible manner for some reason (totally on me for not really knowing how to customize npm modules).

Could such a toggle be made, or could you show me how to implement this myself perhaps?

Thanks a lot!

**Comment by ThomasFKJorna** [comment]
Yes, and it didn't work before, but I tried a little harder and it now works! I still feel like it might be nice to have the option to do it through uniorg (feels somehow "wasteful" to reparse the file) but the performance gain is probably so minimal to not make it worth adding a separate parameter for it.
Thanks again!

**Comment by rasendubi** [comment]
Yeah, I agree. rehype-section also seems to be unmaintained and will likely break when we finally migrate to ESM-only. It’s good enough as a workaround now, but I will leave this issue open and add a flag to uniorg sometime soon.
